tests/runfiles/linux.run

```diff
@@ -98,13 +98,20 @@ tests = ['zfs_create_001_pos', 'zfs_create_002_pos', 'zfs_create_003_pos',
     'zfs_create_013_pos']
 
 # DISABLED:
+# zfs_destroy_001_pos - busy mountpoint behavior
 # zfs_destroy_005_neg - busy mountpoint behavior
 [tests/functional/cli_root/zfs_destroy]
+<<<<<<< 848a24ecacb5cb97541d53b6d5d55b947cce011d
```

Comment:
@jsalinasintel this merge conflict needs to be resolved.  Since the patch revolves around the `online_offline` there shouldn't need to be any changes here.